src/kubernetes/server/server-config.yaml

```diff
@@ -37,6 +37,13 @@ data:
               kubernetes:
                 accounts:
                   default:
+                    secretKeyRefs:
```

Comment:
per the docs above this would only get used if you enable the flag -- given that seems like these shouldn't be included by default?

Reply:
Updated.